django_ninja_crudl/crudl.py

```
@@ -30,381 +33,114 @@
-        if isinstance(
-            field,
-            models.AutoField | models.BigAutoField | models.SmallAutoField,
-        ):
-            return "int"
-        if isinstance(field, models.UUIDField):
-            return "uuid"
-        msg = f"Unknown primary key type: {field}"
-        raise TypeError(msg)
-
-    def __new__(cls, name: str, bases: tuple[type, ...], dct: dict[str, Any]) -> type:
+    def __new__(
```

Ruff (0.8.2)

`__new__` is too complex (56 > 10)

(C901)